docs/compose/treafik/docker-compose.yml

```diff
@@ -0,0 +1,139 @@
+version: '2'
+
+services:
+
+  # This would normally not be here, but where you define your system services
+  traefik:
+    image: traefik:alpine
+    command: --docker
+    restart: always
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock"
+      - "/data/traefik/acme.json:/acme.json"
+      - "/data/traefik/traefik.toml:/traefik.toml"
+      # This may be needed (plus defining mailu_default external: true) if traefik lives elsewhere
+      # networks:
+      # - mailu_default
+
+  # The certdumper is external, but adapted for using traefik with mailu
+  certdumper:
+    restart: always
+    image: nebukadneza/traefik-certdumper:latest
+    environment: # Make sure this is the same as the main=-domain in traefik.toml
+      - DOMAIN=mail.your.doma.in
```

Comment:
We already use `DOMAIN` for the primary mail domain. Although this does overwrite that, it will lead to confusion. I have a comment lower on this page to use something like `TRAEFIK_HOST` in `.env`. That would also take care of passing it down to the container.